includes/class-fs-logger.php

```diff
@@ -329,7 +348,7 @@ public static function _set_storage_logging( $is_on = true ) {
 				 *
 				 * @link https://core.trac.wordpress.org/ticket/2695
 				 */
-				$result = $wpdb->query( "CREATE TABLE {$table} (
+				$result = $wpdb->query( "CREATE TABLE IF NOT EXISTS {$table} (
```

Comment:
Is this `IF NOT EXISTS` needed now that we are dropping the table anyway?

Reply:
Table could exist if it was created with a previous version and never deleted